### DISCUSSION TITLE
如何添加公开访问密码

### DISCUSSION THREAD

**Discussion by 1426239465** [AlistGo/alist#199] [Q&A] [answered]
### Alist Version / Alist 版本

v2.0

### Describe the bug / 问题描述

如何添加公开访问密码

### Reproduction / 复现链接

如何添加公开访问密码

### 日志 / Logs

```shell
如何添加公开访问密码
```

**Comment by xhofe** [comment, marked as answer]
看文档 https://alist-doc.nn.ci/docs/meta/password

另：请不要在issues中提问，讨论与提问请到[discussions](https://github.com/Xhofe/alist/discussions)
